src/scripts/main.js

```diff
@@ -1,3 +1,224 @@
 'use strict';
 
-// write code here
+const DESC = 'desc';
+const ASC = 'asc';
+const HEADERS = ['name', 'position', 'office', 'age', 'salary'];
+const OFFICES = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+const itemCollection = document.querySelectorAll('tbody tr');
+const tableHeader = document.querySelector('thead tr');
+const tableBody = document.querySelector('tbody');
```

Comment:
It's better to move DOM selection operations inside a DOMContentLoaded event handler or inside a function that will be called when the DOM is fully loaded. This is to ensure that the elements you are trying to select are already loaded and present in the DOM.